README.rst

```diff
@@ -33,9 +33,17 @@ Unit tests are also run on CI.
 
 The system tests are run using this command::
 
-  $ nix-build nixos/system-tests.nix
+  $ sudo --preserve-env nix-build nixos/system-tests.nix
 
 The system tests boot QEMU VMs which prevents them from running on CI at this time.
+The build requires > 10 GB of disk space, and the VMs might be timing out on slow or busy machines.
+If you run into timeouts, try `raising the number of retries <https://github.com/PrivateStorageio/PrivateStorageio/blob/e8233d2/nixos/modules/tests/run-introducer.py#L55-L62>`_.
+
+It is also possible go through the testing script interactively - useful for debugging::
+
+  $ sudo --preserve-env nix-build -A private-storage.driver nixos/system-tests.nix
+
+This will give you a result symlink in the current directory.  Inside that is bin/nixos-test-driver which gives you a kind of REPL for interacting with the VMs.  The kind of `Perl in this testScript <https://github.com/PrivateStorageio/PrivateStorageio/blob/78881a3/nixos/modules/tests/private-storage.nix#L180>`_ is what you can enter into this REPL.
```

Comment:
Great.  For what it's worth, the newlines in this doc are placed roughly according to https://rhodesmill.org/brandon/2012/one-sentence-per-line/

This isn't a hard requirement but I find the maintenance consequences pleasing.  If you want to try it out too, great. :)

Reply:
That link is gold! Thanks!